### PR TITLE
Release nodejs-storage v2.0.0

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This pull request was generated using releasetool.

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
Resolves #342

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+bucket.upload('https://example.com/images/image.png', function(err, file, res) {
+  // handle upload...
+});
+```
+
+#### New Code
+
+```js
+const request = require('request');
+let readStream = request(url).createReadStream();
```
Comment (stephenplusplus) [inline CHANGELOG.md]: There isn't a `createReadStream()` method. This should just be:

```js
request(url).pipe(file.createWriteStream());
```

Comment (JustinBeckwith) [reply CHANGELOG.md]: Thank you! Can you check what I have there now?

**Comment by JustinBeckwith** [comment]
Release is at https://github.com/googleapis/nodejs-storage/releases/tag/v2.0.0

**Pull request merged**